core/sys/windows/kernel32.odin

```diff
@@ -21,6 +21,16 @@ COMMON_LVB_REVERSE_VIDEO   :: WORD(0x4000)
 COMMON_LVB_UNDERSCORE      :: WORD(0x8000)
 COMMON_LVB_SBCSDBCS        :: WORD(0x0300)
 
+EV_BREAK :: DWORD(0x0040)
```

Comment:
Can you align these like the constants above?

Reply:
Done. Hope I understood correctly